### DISCUSSION TITLE
How to insert " and ' in one sentence of sql at the same time?

### DISCUSSION THREAD

**Discussion by limeng32** [apache/iotdb#3814] [Q&A]
We know that single quotation marks (') can be inserted in double quotation marks, and double quotation marks (") can be inserted in single quotation marks. However, if an indicator has both single quotation marks (') and double quotation marks (") in the same time, how to insert it?

**Comment by Alima777** [comment]
Can you give one example?

**Comment by limeng32** [comment]
I can Use:
insert into root.ln.wf01.wt01 (timestamp,asd) values(now()," ' ");
To add a single quote record, I can also use:
insert into root.ln.wf01.wt01 (timestamp,asd) values(now(),' " ');
To add a double quote record, but I can’t add a record with both single quotes and double quotes, because there is no escape mechanism;

**Comment by jixuan1989** [comment]
Now you can use "\" as the escape char, but the "\" will not be removed from the values...

![image](https://user-images.githubusercontent.com/1021782/130723117-f67b6fc9-63b7-4c8f-be67-0e9219b364f0.png)

**Reply by limeng32** [reply]
I think you mean "\\", but in this rich-editor you must write '\\\\' to display '\\'.
How about use two single quotation marks '' to escape one single quotation mark ' , just like mysql?
